### PR TITLE
Correct the learning rate as per the code snippet

### PR DESCRIPTION
**Pull request by ptlrs**
The training step, `train_step = tf.train.GradientDescentOptimizer(0.05).minimize(cross_entropy)` states the learning rate as 0.05, whereas the documentation states 0.5. This needs to be corrected.

### CODE REVIEW

**Review by caisq — APPROVED** [review]

**Comment by caisq** [comment]
Thanks, @patelrishabh
